app/src/components/Legend.css

```diff
@@ -27,3 +27,24 @@
   margin: 0px;
   font-weight: bold;
 }
+
```

Comment:
Could this be refactored to styled components, to keep the codebase consistent?

Reply:
It is done